data_and_models/pipelines/ner/spacy_transformers/dvc.yaml

```diff
@@ -0,0 +1,33 @@
+stages:
+  train_biobert_cord19:
+    cmd: >-
```

Comment:
I can imagine that this file could be integrated in the main `ner pipeline` if we decide to go for that solution, right ? At least, it is the way it is going to look in the future ?

Reply:
Yes :)

I have done it like this because that's not clear to me if this PR is more than an experiment or not at the moment. Indeed, it seems the work continues on #329.